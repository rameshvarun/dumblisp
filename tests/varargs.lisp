(defun print-wrapper args
  (print (len args) "arguments supplied.")
  (apply print args))
(print-wrapper 4 5 "hello" FALSE)