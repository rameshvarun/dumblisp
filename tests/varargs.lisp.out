4 arguments supplied.
4 5 hello FALSE
